resources/lib/play_utils.py

```diff
@@ -841,6 +841,13 @@ def external_subs(media_source, list_item, item_id):
             source_id = media_source['Id']
 
             language = stream.get('Language', '')
+            if language and stream['IsDefault']:
+                language = '%s.%s' % (language, 'default')
```

Comment:
```suggestion
                language = '{}.default'.format(language)
```
There's no reason to do string substitution for a constant value.  And personal opinion, I don't like the `%s` syntax.  imo, it's harder to read than using `.format()` and has more oddities in it's behavior.